### DISCUSSION TITLE
Tip for Windows Plot Location: Do NOT Compress Drive

### DISCUSSION THREAD

**Discussion by ddpfy2** [Chia-Network/chia-blockchain#1100] [General]
Hey everyone,

I didn't see a place where this info shown, but I figured it could be added to a Windows Tip for Farmers, like myself. Also, I wanted to be sure I wasn't introducing some anomaly into the system by potentially modifying my plots.

I noticed that when my plots finished and were created on my HDD, the plots were utilizing more disk space than just the files themselves (only plots are on this HDD and no other files). Four (4) plots at ~101GiB were utilizing ~800+GiB on my HDD. I noticed that I had the "Compress this drive to save disk space" checkbox checked for the HDD (see below). Since I was new to the farming process and had low TXCH, I decided to go through with the risk of uncompressing the drive (since I was unaware of the what the outcome would be).

I shut down the Chia GUI, unchecked the "Compress this drive to save disk space" checkbox, let the drive decompress itself over the next handful of hours, noticed the files were closer to the ~404GiB I expected, restarted the Chia GUI, let it sync, and thankfully, the plots were still recognized by the application.

![image](https://user-images.githubusercontent.com/61924995/109428001-8fe59d80-79ba-11eb-957d-be68dd2fd6a4.png)

I hope this helps someone else, and please let me know if you have any questions!

Thanks,
Derek

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
